src/main/java/creativei/service/impl/FollowUpServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package creativei.service.impl;
+
+
+import creativei.dao.FollowUpDao;
+import creativei.entity.FollowUp;
+import creativei.enums.ExceptionType;
+import creativei.exception.DataIntegrityException;
+import creativei.exception.InvalidParamRequest;
+import creativei.service.FollowUpService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service("FollowUpService")
+public class FollowUpServiceImpl implements FollowUpService {
+    @Autowired
+    FollowUpDao followUpDao;
+
+    @Override
+    public FollowUp createFollowUp(FollowUp followUp) throws InvalidParamRequest {
+        if (followUp.getCaseIndex() == null || followUp.getFollowUpDate() == null || followUp.getType() == null || followUp.getStatus() == null || followUp.getSubStatus() == null) {
```

Comment:
Can move this to a private method validateFollowUp(FollowUp followUp). Would be reused in case of update request.

Reply:
moved in a private Boolean method .Issue resolved in 31c4170